src/v/config/configuration.cc

```diff
@@ -3850,6 +3850,14 @@ configuration::configuration()
         model::iceberg_invalid_record_action::drop,
         model::iceberg_invalid_record_action::dlq_table,
       })
+  , iceberg_target_lag_ms(
+      *this,
+      "iceberg_target_lag_ms",
+      "Default value for the redpanda.iceberg.target.lag.ms topic property",
+      {.needs_restart = needs_restart::no, .visibility = visibility::user},
+      // TODO(oren): better default?
+      std::chrono::milliseconds(1min),
```

Comment:
My gut feeling is the default should be a bit higher, at least 5mins or so, just to give flexibility to the scheduler to schedule in more interesting ways rather than just missing deadlines all the time. This needs an empirical evaluation once all the parts are hooked up.

Also purely from a user standpoint my understanding is very few users really want to land data within a minute, my guess is this is typically in hours (I could be wrong here) but the default should be orders of minutes at least.

Reply:
>5mins or so...to give flexibility to the scheduler

makes sense. by the same token, should the minimum value be a couple orders of magnitude higher? not sure how much of a footgun it to allow setting to 10ms, but it doesn't seem like a realistic choice in any case.

Reply:
yes, I agree.

Reply:
for posterity: default is 1m again, minimum is 10s. still arbitrary, but nobody seems very opinionated about this.